javabuilder-authorizer/http_authorizer_function.rb

```diff
@@ -1,109 +1,18 @@
 require 'aws-sdk-lambda'
 require 'jwt'
+require_relative 'jwt_helper'
+include JwtHelper
 
-# The lambda handler takes an event with the query string parameter
-# 'Authorization=token',  where the token is a JWT token generated
-# by dashboard. The handler verifies the token with the appropriate
-# public key (dependant on the environment the request came from),
-# and checks the token has not expired and its issue time is not in the future.
+# The lambda handler takes an event with the query string parameter 'Authorization=token',
```

Comment:
nit: could you add to this documentation (and to the equivalent in the websocket authorizer) to differentiate the two authorizers?

Reply:
Done. Thanks!